zingolib/src/wallet/transaction_context.rs

```diff
@@ -87,23 +87,6 @@ pub mod decrypt_transaction {
             let mut is_outgoing_transaction = false;
             // Collect our t-addresses for easy checking
             let taddrs_set = self.key.get_all_taddrs(&self.config);
-            // Process t-address outputs
-            // If this transaction in outgoing, i.e., we received sent some money in this transaction, then we need to grab all transparent outputs
-            // that don't belong to us as the outgoing metadata
-            // the assumption is either we already decrypted a compact output and filled in some data
-            // or transparent something
-
-            // in the send case, we already know the transaction is outgoing. however, this if clause will not trigger.
-            if self
-                .transaction_metadata_set
-                .read()
-                .await
-                .transaction_records_by_id
-                .total_funds_spent_in(&transaction.txid())
-                > 0
-            {
-                is_outgoing_transaction = true;
-            }
```

Comment:
to save me review time as you have a better understanding of the change here, could you point me to where `is_outgoing_transaction` is set true for shielded if this is removed?